Use the Date from debian/changelog to improve reproducibility of
builds.

Index: u-boot/Makefile
===================================================================
--- u-boot.orig/Makefile
+++ u-boot/Makefile
@@ -1231,10 +1231,14 @@ define filechk_version.h
 	echo \#define LD_VERSION_STRING \"$$($(LD) --version | head -n 1)\"; )
 endef
 
+ifeq ($(SOURCE_DATE_EPOCH),)
+  SOURCE_DATE_EPOCH = $(shell date --utc +%s)
+endif
+
 define filechk_timestamp.h
-	(LC_ALL=C date +'#define U_BOOT_DATE "%b %d %C%y"'; \
-	LC_ALL=C date +'#define U_BOOT_TIME "%T"'; \
-	LC_ALL=C date +'#define U_BOOT_TZ "%z"')
+	(LC_ALL=C date --utc --date "@$(SOURCE_DATE_EPOCH)" +'#define U_BOOT_DATE "%b %d %C%y"'; \
+	LC_ALL=C date --utc --date "@$(SOURCE_DATE_EPOCH)" +'#define U_BOOT_TIME "%T"'; \
+	LC_ALL=C date --utc --date "@$(SOURCE_DATE_EPOCH)" +'#define U_BOOT_TZ "%z"')
 endef
 
 $(version_h): include/config/uboot.release FORCE
